source/css/sass/patterns/atoms/date.scss

```diff
@@ -0,0 +1,3 @@
+.date {
+  @include label-content-typography(inherit);
```

Comment:
We shouldn't set this here. The Date atom could be used in lots of places, when it's used inside ContentMeta it should inherit the right styling anyway.